tools/issue-labeler/src/Azure.Sdk.Labels/Azure.Sdk.LabelTrainer.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
Not essential, but it would be nice to pull out the target framework and throw it into the build props file so that we can bump with a single change.